cordapp-example/workflows-kotlin/src/main/kotlin/com/example/utils/SumByLong.kt

```diff
@@ -3,10 +3,7 @@ package com.example.utils
 inline fun <T> Iterable<T>.sumByLongSecure(selector: (T) -> Long): Long {
     var sum = 0L
     for (element in this) {
-        sum += selector(element)
-        if (sum < 0) {
-            throw RuntimeException()
-        }
+        sum = Math.addExact(sum, selector(element))
     }
     return sum
```

Comment:
The one declared in `contracts-kotlin` should be accessible here. No need to redeclare.